src/routes/uonetplus-uczenplus.ts

```diff
@@ -0,0 +1,152 @@
+import { Request, Response, Router } from 'express';
+import protocol from '../utils/connection';
+
+const router = Router();
+
+router.get('/', (req: Request, res: Response) => {
+  const base = protocol(req) + '://' + req.get('host') + '/powiatwulkanowy/123456';
+  res.json({
+    loginEndpoint: base + '/LoginEndpoint.aspx',
+    app: base + '/App',
+    api: [
+      base + '/api/Context',
+      base + '/api/Cache',
+      base + '/api/OkresyKlasyfikacyjne',
+      base + '/api/Zebrania',
+      base + '/api/SprawdzianyZadaniaDomowe',
+      base + '/api/SprawdzianSzczegoly',
+      base + '/api/ZadanieDomoweSzczegoly',
+      base + '/api/Uwagi',
+      base + '/api/Frekwencja',
+      base + '/api/Oceny',
+      base + '/api/Nauczyciele',
+      base + '/api/Informacje',
+      base + '/api/NieprzeczytaneWiadomosci',
+      base + '/api/DostepOffice',
+      base + '/api/ZarejestrowaneUrzadzenia',
+      base + '/api/PodrecznikiLataSzkolne',
+      base + '/api/SzczesliwyNumerTablica',
+      base + '/api/WazneDzisiajTablica',
+      base + '/api/WychowawcyTablica',
+      base + '/api/RealizacjaZajec',
+      base + '/api/PlanZajec',
+      base + '/api/DniWolne',
+    ].sort(),
+  });
+});
+
+router.all('/LoginEndpoint.aspx', (req: Request, res: Response) => {
+  res.redirect(protocol(req) + '://' + req.get('host') + '/powiatwulkanowy/123456/App');
+});
+
+router.all('/App', (_req: Request, res: Response) => {
```

Comment:
Why did you named this request and other bellow in this file `_req` instead of `req`?

Reply:
This is as in the original `.js` file
It maybe to avoid the linter's no-unused-var hint/warning only